pincer/objects/guild/guild.py

```diff
@@ -474,19 +478,340 @@ async def modify_member(
             Whether the member is deafened |default| :data:`None`
         channel_id : Optional[:class:`~pincer.utils.snowflake.Snowflake]
             Voice channel id to move to |default| :data:`None`
+        reason : Optional[:class:`str`]
+            audit log reason |default| :data:`None`
+            
         Returns
         -------
         :class:`~pincer.objects.guild.member.GuildMember`
             The new member object.
         """
         ...
 
-    async def modify_member(self, _id: int, **kwargs) -> GuildMember:
+    async def modify_member(self, _id: int, reason=None, **kwargs) -> GuildMember:
         data = await self._http.patch(
-            f"guilds/{self.id}/members/{_id}", data=kwargs
+            f"guilds/{self.id}/members/{_id}",
+            data=kwargs,
+            headers=remove_none({"X-Audit-Log-Reason": reason})
         )
         return GuildMember.from_dict(construct_client_dict(self._client, data))
 
+      
+    @overload
+    async def create_channel(
+        self,
+        *,
+        name: str,
+        type: Optional[ChannelType] = None,
+        topic: Optional[str] = None,
+        bitrate: Optional[int] = None,
+        user_limit: Optional[int] = None,
+        rate_limit_per_user: Optional[int] = None,
+        position: Optional[int] = None,
+        permission_overwrites: Optional[List[Overwrite]] = None,
+        parent_id: Optional[Snowflake] = None,
+        nsfw: Optional[bool] = None
+    ) -> Channel:
+        """|coro|
+        Create a new channel object for the guild.
+
+        Parameters
+        ----------
+        name : str
+            channel name (1-100 characters)
+        type : Optional[:class:int`]
+            the type of channel
+        topic : Optional[:class:str`]
+            channel topic (0-1024 characters)
+        bitrate : Optional[:class:`int`]
+            the bitrate (in bits) of the voice channel (voice only)
+        user_limit : Optional[:class:`int`]
+            the user limit of the voice channel (voice only)
+        rate_limit_per_user : Optional[:class:`int`]
+            amount of seconds a user has to wait before sending another message (0-21600)
+            bots, as well as users with the permission manage_messages or manage_channel, are unaffected
```

Comment:
Split lines (89 & 104)
```suggestion
            amount of seconds a user has to wait
            before sending another message (0-21600)
            bots, as well as users with the permission
            manage_messages or manage_channel, are unaffected
```